src/external/atoms_calculators.jl

```diff
@@ -69,12 +71,20 @@ AtomsCalculators.@generate_interface function AtomsCalculators.potential_energy(
         system::AbstractSystem, calculator::DFTKCalculator; state = DFTKState(),
         kwargs...)
     compute_scf!(system, calculator, state)
-    calculator.state.scfres.energies.total
+    calculator.state.scfres.energies.total * u"hartree"
 end
 
 AtomsCalculators.@generate_interface function AtomsCalculators.forces(
         system::AbstractSystem, calculator::DFTKCalculator; state = DFTKState(),
         kwargs...)
     compute_scf!(system, calculator, state)
-    compute_forces_cart(calculator.state.scfres)
+    compute_forces_cart(calculator.state.scfres) * u"hartree/bohr"
+end
+
+AtomsCalculators.@generate_interface function AtomsCalculators.virial(
+        system::AbstractSystem, calculator::DFTKCalculator; state = DFTKState(),
```

Comment:
We don't put spaces around `=` in Keyword arguments (same above, actually, which I did not catch last time)